common/persistence/src/main/java/app/coronawarn/datadonation/common/persistence/domain/metrics/TestResultMetadata.java

```diff
@@ -71,7 +76,7 @@
   /**
    * Constructs an immutable instance.
    */
-  public TestResultMetadata(Long id, Integer testResult, Integer hoursSinceTestRegistration,
+  public TestResultMetadata(Long id, Integer testResult, Integer hoursSinceTestRegistration, //NOSONAR parameter no
       Integer riskLevelAtTestRegistration,
```

Comment:
This class does not have equals and hash methods, maybe we can add them... what do you think?

Reply:
makes sense. added in.